guardian/live-app-versions
# Attempt to fix logging

## What does this change?

The logging for this repo is broken - we can't see any custom logs for the last two weeks (and most likely longer) in Kibana. We've taken inspiration from https://github.com/guardian/mobile-save-for-later/pull/112 to replace log4j with slf4j-simple. 

## How to test

We [deployed](https://riffraff.gutools.co.uk/deployment/view/4ac5ea64-9b5a-4e9d-9ac6-b212d4b9eeaf) the build to PROD and can see in [Kibana](https://logs.gutools.co.uk/s/mobile/goto/7f1bb4e0-07d7-11ef-a06c-911a16951718) that logging is now being sent as expected:
 
![image](https://github.com/guardian/live-app-versions/assets/23078809/69b6fc74-c40b-4c5c-b915-08b640ee3f70)
